### ISSUE TITLE
[FE] Fixed A11y linting warnings

### ISSUE THREAD

**Issue by mdewey** [department-of-veterans-affairs/va.gov-team#35976]
## Related Documentation

> these should be links to feature docs

- [Sketch](https://www.sketch.com/s/e79a827e-42cf-4a82-b554-874c75b5c70e)
- [Product Docs](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin)
- [Slack Thread](https://dsva.slack.com/archives/C5HP4GN3F/p1643135144032200)

## Description

> this should be an understandable description of the ticket. This description should have enough context meaning that non-technical team members should get the idea.

The platform team is turning on new a11y linting rules. We have a few violations that need to be addressed

## Tasks

> this is a list of technical tasks that need for this ticket

- [ ] Identify any issues in the `check-in` and `health-care-questionaires` folders
- [ ] Work with the correct teams to fix those erors
- [ ] Create or update unit tests
- [ ] Create or update e2e tests
- [ ] Update relevant documentations
  - [ ] README
  - [ ] Use Cases

## Acceptance Criteria

> when is the ticket done

- [ ] %Add any additional Acceptance Criteria%
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature

**Comment by mdewey** [comment]
Verified and looks good to me
